src/pager.c

```diff
@@ -1429,6 +1430,20 @@ char *supplemental_name;
         if (*dbase_str == ' ')
             ++dbase_str;
     }
+
+    /* remove material type */
+    int l;
+    for (i = 1; i <= NUM_MATERIAL_TYPES; i++) {
+        l = strlen(materialnm[i]);
+        Strcpy(buf, materialnm[i]);
+        Strcat(buf, " ");
+        if (!strncmp(dbase_str, buf, ++l)) {
+            Strcpy(material, buf);
+            dbase_str += l;
+            break;
+        } 
+    }
+
```

Comment:
Note that this should come after the towel handling. I don't think non-cloth towels are possible, but if they were, they'd render as "wet [material] towel".